pkg/registry/core/rest/storage_core.go

```diff
@@ -179,50 +175,90 @@ func (c LegacyRESTStorageProvider) NewLegacyRESTStorage(restOptionsGetter generi
 
 	serviceRest := service.NewStorage(serviceRegistry, endpointRegistry, ServiceClusterIPAllocator, ServiceNodePortAllocator, c.ProxyTransport)
 
-	restStorageMap := map[string]rest.Storage{
```

Comment:
Why not? Core is too broad. I want to disable just ReplicationControllers for example.
runtime-config allows users to disable groups, group versions or specific resources.
It will be confusing if some groups allow disabling specific resources while some groups do not.
I think this group not allowing disabling specific resources was actually a bug.